src/api.v2/controllers/experimentController.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/prefer-default-export */
+const config = require('../../config');
+
+const experiment = require('../model/experiment');
+const userAccess = require('../model/userAccess');
+
+const AccessRole = require('../../utils/enums/AccessRole');
+
+const { OK } = require('../../utils/responses');
+
+const createExperiment = async (req, res) => {
+  const { params: { experimentId }, user, body } = req;
+
+  const { name, description } = body;
+
+  await experiment.create({ id: experimentId, name, description });
+
+  await Promise.all([
+    userAccess.create(
+      { user_id: user.sub, experiment_id: experimentId, access_role: AccessRole.OWNER },
+    ),
+    userAccess.create(
+      { user_id: config.adminArn, experiment_id: experimentId, access_role: AccessRole.OWNER },
+    ),
+  ]);
+
+  res.json(OK());
```

Comment:
what about handling the cases when create fails? Should we handle exceptions in here or in `moelGuts.js`?

Reply:
I think I would look into exceptions handling in a different ticket, we didn't have a clear exceptions handling policy in dynamodb in v1 so I think defining that now would take some extra work. Also on its own it doesn't help too much unless we check the different types of exceptions in the UI and display different errors for each kind of error.